com.microsoft.AzureStorageExplorer.yaml

```diff
@@ -0,0 +1,84 @@
+---
+app-id: com.microsoft.AzureStorageExplorer
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+command: StorageExplorer
+separate-locales: false
+only-arches:
+  - x86_64
+failure: test
+tags:
+  - proprietary
+build-options:
+  append-path: "/usr/lib/sdk/dotnet6/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+  env:
+    PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--share=network"
+  - "--device=dri"
+  - "--talk-name=org.freedesktop.secrets"
+  - "--talk-name=org.gnome.SessionManager"
+  - "--talk-name=com.canonical.AppMenu.Registrar"
+  - "--talk-name=com.canonical.AppMenu.Registrar.*"
+  - "--system-talk-name=org.freedesktop.login1"
+  - "--persist=.IdentityService"
+  - "--persist=ScriptedHost"
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/share/pkgconfig"
+  - "/share/aclocal"
+  - "/man"
+  - "/share/man"
+  - "/share/gtk-doc"
+  - "/share/doc"
+  - "*.la"
+  - "*.a"
```

Comment:
```suggestion
```

libsecret has it's own cleanup, you aren't building anything that needs to be cleaned up.